go_logging_server/logging/setup.go

```
@@ -151,3 +154,19 @@ func createLogFile(fileName string, path string) (*os.File, error) {
 	return file, nil
 
 }
+
+/*Function to handle*/
+func refreshLogsFiles() {
+	c := cron.New()
+
+	//Call refresh of log files
+	_, err := c.AddFunc("00 0 * * *", func() {
+		createLoggingFiles()
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)